docs/markdown/snippets/exe_wrapper_for_cross_built_tests.md

```diff
@@ -0,0 +1,8 @@
+## Test scripts are given the exe wrapper if needed
+
+Meson will now set the `MESON_EXE_WRAPPER` as the `' '` joined command of the
+exe wrapper. This allow wrapper scripts to run test binaries, instead of just
+skipping.
```

Comment:
This should probably be using `shlex.join` instead to make those Windows file names with spaces in them work.

Reply:
Updated to use `mesonlib.join_args` (which is more correct for windows than `shlex.join`)